src/util/native_fs_utils.js

```diff
@@ -419,6 +418,20 @@ async function get_user_by_distinguished_name({ distinguished_name }) {
     }
 }
 
+async function get_anonymous_account_by_name(anon_account_name) {
```

Comment:
The name is constant (`config.NC_ANON_ACCOUNT_NAME`), maybe you can remove this argument from the function signature.